### ISSUE TITLE
Soundflowerbed missing from Soundflower folder

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#92]
```
What steps will reproduce the problem?
1. Installed Soundflower-1.5.2 on SnowLeopard (OS X 10.6.6)

2. Changed permissions on the Soundflower folder so I could open it.

3. The Soundflower folder contains "Soundflowerbed README.rtf" but NOT the 
Soundflowerbed application itself

What is the expected output? What do you see instead?

I had expected to find Soundflowerbed in the folder - the description of the 
download is "Soundflower 1.5.2 Installer (includes SoundflowerBed)"

I also checked (using Terminal) whether the app was there but hidden. It isn't.


What version of the product are you using? On what operating system?

1.5.2 on SnowLeopard 10.6.6


Please provide any additional information below.

I think I've covered it all above.

```

Original issue reported on code.google.com by `john.ray...@gmail.com` on 26 Apr 2011 at 3:39

**Comment by GoogleCodeExporter** [comment]
```
Same Problem.

I hit upon a rather odd workaround (quite by accident)

I downloaded 1.5.1, which also doesn't install soundflowerbed BUT I noticed 
that soundflower is an mpkg file, rather than the pkg file in 1.5.2.  You can 
"show package contents" and navigate to the Content/Packages directory, then 
run sunflowerbed.pkg directly from there, which installs it in the expected 
location - no muss, no fuss :)

the 1.5.2 soundflower + 1.5.1 soundflowerbed seems to work fine (ie: as 
expected).  I just use it so that I can monitor the audio in real time.



```

Original comment by `thoen...@gmail.com` on 26 May 2011 at 7:41

**Comment by GoogleCodeExporter** [comment]
```
This problem has been fixed in the just-released version 1.6 release candidate.
```

Original comment by `t...@electrotap.com` on 3 Feb 2012 at 9:03
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I am not so sure about the fix: I downloaded 1.6.6b as installed. Sunflower is 
working as expected, but SFbed is simply absent from the applicationfolder. Mac 
OS 1.8.4.
Repeated unistall and reinstall replicates the problem.
```

Original comment by `philipwe...@gmail.com` on 24 Jul 2013 at 4:42
